app/main/views.py

```diff
@@ -16,11 +19,16 @@ def index():
     req_opt_desc = Descriptor.query.filter_by(
         id=req_opt_desc.descriptor_id
     ).first()
+    options = Descriptor.query.all()
+    options = [o for o in options if len(o.text_resources) == 0 and o.id is not req_opt_desc.descriptor_id]
```

Comment:
essentially we wanna do something like
```
req_opt_id = -1
if req_opt_desc is not None:
   req_opt_id = req_opt_desc.descriptor_id
options = Descriptor.query.all()
options = [o for o in options if len(o.text_resources) == 0 and o.id != req_opt_desc.descriptor_id]
```
is how i'm thinking? but we should test